app/src/main/res/values/strings.xml

```diff
@@ -8,9 +8,9 @@
 
     <!-- Favorites -->
     <string name="button_favorites">Favorites</string>
-    <string name="button_select_favorites">Select favorites</string>
+    <string name="button_select_favorites">Add more</string>
     <string name="hint_no_favorites_yet">Add your favorite apps here\nby clicking the menu button.</string>
-    <string name="hint_modify_order_of_favorites">Modify the order of favorites in the list below (using the sides to scroll).</string>
+    <string name="hint_modify_order_of_favorites">Hold and move to change the order, using the sides to scroll.</string>

```

Comment:
```suggestion
    <string name="hint_modify_order_of_favorites">Hold and move to change the order, use the sides to scroll.</string>
```
Indeed it is simpler like this.
My only remark is that we could say "use" instead of "using" (as both moving a favorite and scrolling cannot be done at the same time, it is one or the other).

Reply:
```suggestion
    <string name="hint_modify_order_of_favorites">Hold and move to change the order, and use the sides to scroll.</string>

```